content/applications/sales/sales/invoicing/proforma.rst

```diff
@@ -7,23 +7,24 @@ notes the kind and quantity of goods, their value, and other important informati
 and transportation charges.
 
 Pro-forma invoices are commonly used as preliminary invoices with a quotation. They are also used
-for customs purposes during importation. They differ from a normal invoice in that they are not a
+during importation for customs purposes. They differ from a normal invoice, in that they are *not* a
 demand (or request) for payment.
 
-Activate the feature
-====================
+Configuration
+=============
 
-In order to utilize pro-forma invoices, the *Pro-Forma Invoice* feature **must** be activated. To do
-that, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and in the
-:guilabel:`Quotations & Orders` section, click the checkbox next to :guilabel:`Pro-Forma Invoice`.
-Then, click :guilabel:`Save` to save all changes.
+In order to utilize pro-forma invoices, the *Pro-Forma Invoice* feature **must** be activated.
+
+To enable this feature, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and
+in the :guilabel:`Quotations & Orders` section, click the checkbox next to :guilabel:`Pro-Forma
```

Comment:
The ampersand does not appear to break the rst, but let's escape it just in case
```suggestion
in the :guilabel:`Quotations \& Orders` section, click the checkbox next to :guilabel:`Pro-Forma
```